socorro/processor/symbol_cache_manager.py

```diff
@@ -14,11 +14,19 @@
 from configman.converters import class_converter, timedelta_converter
 
 from socorro.lib.datetimeutil import utc_now, UTC
-import pyinotify
+if os.uname()[0] == 'Darwin':
```

Comment:
You should make it `!= 'Linux'`, since this won't work on anything else (Windows, BSD, etc)